### PR TITLE
portmgr entry for 2021q3

### PR DESCRIPTION
**Pull request by rene0**

### CODE REVIEW

File: 2021q3/portmgr.adoc [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+The Ports Management Team is responsible for overseeing the overall direction of the Ports Tree, building packages, and personnel matters.
+Below is what happened in the last quarter.
+
+There are currently 46,500 ports in the Ports Tree according to FreshPorts.
+The open PR count for ports is currently 2,588, of which 608 are unassigned.
+The last quarter saw 9,833 commits on the "main" branch by 148 committers and 743 commits on the quarterly branch by 54 committers.
+Compared to 2021q2, activity mostly remained the same, the PR counts increased a bit but there were also more commits to the quarterly branch.
+
+During 2021q3, we welcomed Daniel Engberg (diizzy@) and Yasuhiro Kimura (yasu@), we said goodbye to culot@ and grog@ after their commit bits were taken in for safekeeping.
```
Comment (pauamma) [inline 2021q3/portmgr.adoc:21]: s/we said/and we said/ (or use ; instead of , to avoid a comma splice).

File: 2021q3/portmgr.adoc [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+Two new uses were introduced: angr to help with testing Python-related ports and mlt to help depending on mlt, a multimedia framework for TV broadcasting.
+Ruby saw minor updates for the 2.6, 2.7, and 3.0 series.
+
+The "big" packages were also updated: pkg to 1.17.2, Firefox to 93.0, and Chromium to 92.0.4515.159.
+
+As usual, exp-runs were performed by antoine@ but only those of July were recorded.
+There were 5 exp-runs to test adding CLOCK_*_COARSE compatibility definitions for Linux and to test ports updates.
```
Comment (pauamma) [inline 2021q3/portmgr.adoc:29]: Markdown seems to parse the * as a formatting marker. If AsciiDoc does the same, you need to escape it.

**Comment by rene0** [comment]
> For next time: maybe keep pushes and PRs separate from one quarter to the next?

Hm yes, I actually keep a clone of this repository, so I merge back the upstream changes when starting a new report and then create the pull request. These merge and edit commits are separate, but GitHub combines those when creating the pull request?

**Comment by pauamma** [comment]
> > For next time: maybe keep pushes and PRs separate from one quarter to the next?
> 
> Hm yes, I actually keep a clone of this repository, so I merge back the upstream changes when starting a new report and then create the pull request. These merge and edit commits are separate, but GitHub combines those when creating the pull request?

I think that's standard git (and therefore GitHub) behavior if all changes are on the same branch.

**Comment by pauamma** [comment]
@debdrup, are you OK with me merging directly? I don't want to mess with your workflow.

**Comment by debdrup** [comment]
> @debdrup, are you OK with me merging directly? I don't want to mess with your workflow.

I think I've mentioned this elsewhere, but I'm fine with you merging any report entries you address, once you're satisfied with them and have approved them.

**Comment by debdrup** [comment]
Yep, this has been merged.

**Pull request closed without merging**